integrationTests/multiShard/block/executingMiniblocksSc_test.go

```diff
@@ -38,10 +44,15 @@ func TestProcessWithScTxsTopUpAndWithdrawOnlyProposers(t *testing.T) {
 	advertiserAddr := integrationTests.GetConnectableAddress(advertiser)
 
 	nodeShard0 := integrationTests.NewTestProcessorNode(maxShards, 0, 0, advertiserAddr)
+	nodeShard0.EconomicsData.SetMinGasPrice(0)
+
 	nodeShard1 := integrationTests.NewTestProcessorNode(maxShards, 1, 1, advertiserAddr)
+	nodeShard1.EconomicsData.SetMinGasPrice(0)
+
 	hardCodedSk, _ := hex.DecodeString("5561d28b0d89fa425bbbf9e49a018b5d1e4a462c03d2efce60faf9ddece2af06")
 	nodeShard1.LoadTxSignSkBytes(hardCodedSk)
 	nodeMeta := integrationTests.NewTestProcessorNode(maxShards, sharding.MetachainShardId, 0, advertiserAddr)
+	nodeMeta.EconomicsData.SetMinGasPrice(0)
```

Comment:
Can we move the setMinGasPrice to intetgrationTests.NewTestProcessorNode instead?
Then there's no need to do this separately after each test node creation.

If we want to test also with different gas prices, we can give this as parameter to NewTestProcessorNode (in this case maybe also change from parameter list to argument struct)

Reply:
This part of code was removed and global variable for min gas price is set to zero.